_config.yml

```diff
@@ -107,9 +107,9 @@ visa_email: "visas@djangocon.eu"
 # Links
 cfp_application: "https://pretalx.com/djangocon-europe-2023/cfp"
 hotel_reservation_link: "https://www.marriott.com/events/start.mi?id=1642551494496&key=GRP"
-# slack_link: "https://djangoconus2023.herokuapp.com"
 opportunity_grant_application: "https://docs.google.com/forms/d/1yc1HqqcJwi669vxTTs1daJbTeiNYwqM2h3gIgqbpWVE/viewform"
 slack_link: ""
+# slack_link: "https://join.slack.com/t/djangoconeurope/shared_invite/zt-1gjg5lqkz-qVQkNnhjztXVme7TQ7ziQA"
```

Comment:
This is the 2022 DjangoCon Europe link. I’ve copied it here so we have an easier time figuring out where to update it once needed.